packages/nextjs/hooks/scaffold-eth/useScaffoldEventHistory.ts

```diff
@@ -38,6 +38,7 @@ export const useScaffoldEventHistory = <
 }) => {
   const [events, setEvents] = useState(<any>[]);
   const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState(<string>"");
```

Comment:
No need to use assertion here, since `""` is already of type `string`.

Personally, I prefer to let states be `undefined` rather than "empty", otherwise there are cases where it's difficult to distinguish between an actual empty array/string and a state that is waiting for an async proceess to complete.

So I'd go for this:
```typescript
const [error, setError] = useState<string>();
```

That tells the type it will be `string | undefined`.

Same comment applies to line 39, where I'd go for

```typescript
const [events, setEvents] = useState<any[]>();
```

Reply:
Changed! Thanks!